leasing/models/rent.py

```diff
@@ -995,7 +995,11 @@ class LeaseBasisOfRent(ArchivableModel, TimeStampedSafeDeleteModel):
     area = models.DecimalField(verbose_name=_("Area amount"), decimal_places=2, max_digits=12)
 
     # In Finnish: Vyöhyke
-    zone = models.PositiveIntegerField(verbose_name=_("Zone"), null=True, blank=True)
+    zone = models.PositiveIntegerField(verbose_name=_("Zone"), null=True, blank=True, choices=(
```

Comment:
Is there a good reason this is not an EnumField?

Reply:
Agreed, I'd rather use an EnumField here since most other choiced fields use enums as well.